WordLadder/src/assignment3/Node.java

```diff
@@ -0,0 +1,27 @@
+package assignment3;
+
+import java.util.*;
+
+public class Node {
+	
+	public String word;
+	public Node previous;
+	
+	public Node (String m, Node p){
+		this.word = m;
+		this.previous = p;
+	}
+	
+	public ArrayList<String> createNodeList(){
+		ArrayList<String> list = new ArrayList<String>();
+		list.add(this.word);
+		while(previous != null){
+			list.add(previous.word);
+			previous = previous.previous;
+		}
+		
+		Collections.reverse(list);
```

Comment:
The Reason this is reversed is so because the node starts from the end then goes to the beginning. Therefore the resulting list would be reversed.